core/src/main/java/greencity/controller/EcoNewsController.java

```diff
@@ -175,8 +175,6 @@ public ResponseEntity<EcoNewsGenericDto> update(
     @ApiOperation(value = "Get eco news by id.")
     @ApiResponses(value = {
         @ApiResponse(code = 200, message = HttpStatuses.OK),
-        @ApiResponse(code = 400, message = HttpStatuses.BAD_REQUEST),
```

Comment:
It should be possible to get a 400 response when someone tries to enter char as id

Reply:
https://github.com/ita-social-projects/GreenCity/issues/4547

This issue says that 404 is also redundant but used in the if statement in the implementation (so I left that code).

So does this ticket make any sense?

Reply:
Fixed